frontend/src/components/entry-list-item/helpers/buttonlHandlers.ts

```diff
@@ -0,0 +1,21 @@
+import {apiCalls} from "../../../services/api/apiOperations.ts";
+import type {Dispatch, SetStateAction} from "react";
+import type {IMeal} from "../../../models/meal/IMeal.ts";
+
+export const buttonHandlers = {
+    handleDelete: async (name:string,cb:Dispatch<SetStateAction<IMeal[]>>) =>{
```

Comment:
Please, keep all indentations and spaces according to the following style across the project:
```suggestion
    handleDelete: async (name: string, cb: Dispatch<SetStateAction<IMeal[]>>) => {
```
It gets much cleaner this way